product_docs/docs/biganimal/release/overview/03_security/01_securing_your_cluster.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Customizing policy definitions"
+---
+
+For each subscription belonging to your organization, Azure automatically assigns a default set of policies to ensure baseline security of your resources. You can customize these default policies to match BigAnimal's specific requirement and further harden security of your resources. 
+
+### Security configurations required by BigAnimal
+
+Use the values below each of the parameters to configure the default ASC policy of a subscription. 
+
+!!! Note
+    
+    You can directly copy the parameter values provided below. Values are provided in JSON wherever applicable.
+
+##### Allowed container ports list:
```

Comment:
These should all be connected to specific ASC recommendations, eh?

Reply:
i have asked @liudanbj2021 to provide a mapping of the parameters with the alerts.

Reply:
This is resolved.